amp/TEMPLATE/reampv2/src/modules/geocoder/components/AppContext.jsx

```diff
@@ -22,13 +23,13 @@ class AppContext extends Component {
 
     componentDidMount() {
         this.props.fetchTranslations(defaultTrnPack);
-        this.props.loadActivities();
+        this.props.loadGeocoding();
     }
 
     render() {
         return this.props.translationPending
             ? (<Loading/>) :
```

Comment:
it is usually a good practice to use destructuring assignment when using state props or context. See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md for more info